app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -509,6 +509,7 @@ class ParkingViewModel(
     // This coroutine will update the closest parkings when all the tiles have been fetched. Note
     // that we don't need to apply the filter again since we use the filteredRectParkings flow.
     viewModelScope.launch {
+      Log.e("ParkingViewModelCenter", "${_circleCenter.value}")
```

Comment:
Same here, we cam keep it if you want but we should use Log.d rather than Log.d

Reply:
Yes this one was and is still useful so I will pass it to Log.d ! Thanks for pointing this out !